### ISSUE TITLE
Fishy PayPal script

### ISSUE THREAD

**Issue by Shaftoe62** [jeanropke/RDR2CollectorsMap#222]
The PayPal form element you are using is behaving rather badly.

There is an image missing (console error) and also it sends me to a donation page in Brasilian(?).

Also the (Chrome) console gives a cookie warning about the PayPal implementation. 

Code reference: index.html line 288 to line 296

![image](https://user-images.githubusercontent.com/53898212/66773819-40cfc300-eec0-11e9-803a-d986f649b4ef.png)

**Comment by Rakmarok** [comment]
> There is an image missing (console error) and also it sends me to a donation page in Brasilian(?).

Jean Ropke lives in Brazil. The rest is above my knowledge.

**Comment by iligalodin** [comment]
Image missing? Hmm cause we don’t want to clutter the menu, brazilian language? Jean? Where you from

**Comment by jeanropke** [comment]
I'm using the Paypal code.
I'm living in Brazil.

**Comment by Shaftoe62** [comment]
The code is "shaky". Should not be in Portugese for me (i'm in the Netherlands). And it gives a lot of errors in the console.

**Comment by Shaftoe62** [comment]
The hard-coded Locale should be ... well, local of course.

**Comment by Shaftoe62** [comment]
google "javascript navigator.languages"

**Comment by jeanropke** [comment]
Why? Where is the locale? The 404 image? This image? https://www.paypal.com/en_US/i/scr/pixel.gif

**Comment by jeanropke** [comment]
And this warning about cookies is everywhere
![samesite](https://user-images.githubusercontent.com/6243818/66791013-a922a280-eec8-11e9-95d3-5be499af9da2.png)

![samesite2](https://user-images.githubusercontent.com/6243818/66791055-e129e580-eec8-11e9-9d8c-91a5706702c8.png)

**Comment by Shaftoe62** [comment]
in index.html:

![image](https://user-images.githubusercontent.com/53898212/66791255-ad17ea00-eef3-11e9-9160-f195d72f72f6.png)

**Comment by jeanropke** [comment]
Its an image, not a locale

**Comment by Shaftoe62** [comment]
Well, anyway I get redirected to:

https://www.paypal.com/donate/?token=bTC2DKT0g2bAIpmBz9XdBz8b9Xk3pd2duq9sdDoQIs5rbF9hNeHwm-e74sqX9_L0cTF1IW&country.x=BR&locale.x=

And that could also be:

https://www.paypal.com/donate/?token=bTC2DKT0g2bAIpmBz9XdBz8b9Xk3pd2duq9sdDoQIs5rbF9hNeHwm-e74sqX9_L0cTF1IW&country.x=BR&locale.x=EN

**Comment by Shaftoe62** [comment]
@jeanropke  Somehow the script at https://www.paypal.com/cgi-bin/webscr sets the language to Brasilian. I'm looking into it. If I can find a working solution I will let you know.

**Comment by Shaftoe62** [comment]
Ok, I've figured it out. Rather tricky because the browser immediately stores the locale in a cookie. So I had to try with different browsers. 

Included html file has comments that explain what happens.

This forces the paypal script to redirect to:

https://www.paypal.com/donate/?token=m8mQuZGdSEEy2jhbWVYn3v_h5opUoqN-PsWwSfCtPQRw8NjyQh_1tg-rs1-9y-Ku7DmFQW&country.x=US&locale.x=US&Z3JncnB0=

To test it you might want to use a different browser or 'destroy' you current session. 
[paypaltest.html.txt](https://github.com/jeanropke/RDR2CollectorsMap/files/3727047/paypaltest.html.txt)
